python/masonite/requirements.txt

```diff
@@ -1,3 +1,3 @@
-masonite>=3.0,<3.1
+masonite>=4.0,<4.1
```

Comment:
Should be >=4,<5

Masonite is now SEMVER

Reply:
I think we rather use minor range
+ all frameworks use it (semver or not)
+ patch level is something too complicated to maintain / meaningless to expose